docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,179 @@
+# Allowing OpenShift Groups to be Synced With Sources on an LDAP Server and Making it Extensible for Other External Sources
```

Comment:
There is no use case today that requires sync to something beyond LDAP.  There *are* other sources but no one has ever asked us to sync to them.